roles/identity-management/manage-rh-sso-identities/tasks/main.yml

```diff
@@ -8,4 +8,16 @@
   when:
     - identities.targets is undefined or 'rh-sso' in identities.targets
   tags:
-    - always
+    - create-user
+    - delete-user
+
+- name: Process all tasks
```

Comment:
Do we need to handle group creation prior to user creation? Otherwise our groups won't exist when the user is getting created.